examples/delete_default_gateway_routes/main.tf

```diff
@@ -14,14 +14,6 @@
  * limitations under the License.
  */
 
-provider "google" {
```

Comment:
@paulpalamarchuk Why are you removing these? We specifically do want them.

Reply:
Moved to the `versions.tf`.

Reply:
I have no idea why you removed these. They're exactly right as they are. Please do not change anything about this line, this is exactly how provider versions should be declared in examples.